lib/internal/errors.js

```diff
@@ -858,6 +859,22 @@ const genericNodeError = hideStackFrames(function genericNodeError(message, erro
   return err;
 });
 
+ObjectDefineProperty(AggregateErrorPrototype, SymbolFor('nodejs.util.inspect.custom'), {
+  __proto__: null,
+  enumerable: false,
+  writable: false,
+  value: function inspect(depth, options) {  // eslint-disable-line func-name-matching
+    if (depth < 0)
+      return this;
+    const opts = {
+      ...options,
+      depth: options.depth == null ? null : options.depth - 1
```

Comment:
Can't `options` be `undefined`?
```suggestion
      depth: options?.depth == null ? null : options.depth - 1
```